pkgs/servers/nextcloud/packages/apps/memories-paths.diff

```diff
@@ -0,0 +1,31 @@
+diff --git a/lib/Settings/SystemConfig.php b/lib/Settings/SystemConfig.php
+index 7814f071..bb1309fa 100644
+--- a/lib/Settings/SystemConfig.php
++++ b/lib/Settings/SystemConfig.php
+@@ -124,6 +124,13 @@ class SystemConfig
+      */
+     public static function get(string $key, mixed $default = null): mixed
+     {
++        switch ($key) {
++          case "memories.exiftool": return "@exiftool@";
++          case "memories.vod.ffmpeg": return "@ffmpeg@";
++          case "memories.vod.ffprobe": return "@ffprobe@";
++          case "memories.vod.path": return "@go-vod@";
++        }
++
+         if (!\array_key_exists($key, self::DEFAULTS)) {
+             throw new \InvalidArgumentException("Invalid system config key: {$key}");
+         }
+@@ -154,6 +161,12 @@ public static function get(string $key, mixed $default = null): mixed
+      */
+     public static function set(string $key, mixed $value): void
+     {
++        // Ignore those paths and always use the nix paths.
++        // We cannot return a proper error message except a 500 here without changing the code to much.
++        if (in_array($key, array("memories.exiftool", "memories.vod.ffmpeg", "memories.vod.ffprobe", "memories.vod.path"))) {
++            return;
```

Comment:
```suggestion
+            throw new \InvalidArgumentException("Cannot set nix-managed key: {$key}");
```

This way you can get it in the logs at least:

![image](https://github.com/user-attachments/assets/cbb47a93-144e-4b69-8b1d-999dc3660fa7)

Reply:
I really do not want to do that and fill the log app with more logs that are not actionable. I already have way to much noise in there which just exists.

Reply:
There is a small popup

![image](https://github.com/user-attachments/assets/18a862f8-8a6d-48c1-a818-0da3f1fc902f)

Reply:
Was that pop up always there? I am not sure anymore but it is better to fail loud than quiet.